inji-verify-test/pom.xml

```diff
@@ -1,62 +1,128 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>io.mosip.uiautomation</groupId>
-  <artifactId>inji-verify-test</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
-
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cucumber.version>7.17.0</cucumber.version>
-  </properties>
-
-	<dependencies>
-		<dependency>
-			<groupId>com.microsoft.playwright</groupId>
-			<artifactId>playwright</artifactId>
-			<version>1.43.0</version>
-		</dependency>
-		<dependency>
-			<groupId>io.cucumber</groupId>
-			<artifactId>cucumber-java</artifactId>
-			<version>${cucumber.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>io.cucumber</groupId>
-			<artifactId>cucumber-junit</artifactId>
-			<version>${cucumber.version}</version>
-			<scope>test</scope>
-		</dependency>
-		<dependency>
-			<groupId>io.cucumber</groupId>
-			<artifactId>cucumber-testng</artifactId>
-			<version>${cucumber.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>io.cucumber</groupId>
-			<artifactId>cucumber-picocontainer</artifactId>
-			<version>${cucumber.version}</version>
-		</dependency>
-		<!-- https://mvnrepository.com/artifact/org.json/json -->
-		<dependency>
-			<groupId>org.json</groupId>
-			<artifactId>json</artifactId>
-			<version>20240303</version>
-		</dependency>
-
-	</dependencies>
-
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.10.1</version>
-				<!-- References to interface static methods are allowed only at source level 1.8 or above -->
-        <configuration>
-          <source>21</source>
-          <target>21</target>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+<project xmlns="http://maven.apache.org/POM/4.0.0"

```

Comment:
What is the general convention we are following for the folder name for the test rig ?
I see the apitest folder name used in the below repos, but here I can see it as inji-verify-test.

https://github.com/mosip/esignet/tree/develop/apitest
https://github.com/mosip/mimoto/tree/develop/apitest

Reply:
apitest folder name is used for API automation tests.
Here, it is a UI automation test, so we will maintain as inji-verify-test.

Reply:
Provided info

Reply:
In the similar lines, should we not call it as ui-test instead of using the module name ? Someone see this name will they not get confused if that is api test or ui test ?

Reply:
We can follow the same.. to make sure that we have uniformity.. let us follow the ui-test for other modules as well

Reply:
Noted @lsivanand , we will update the PR

Reply:
Resolved